incubator/hydra/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ include "hydra.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "hydra.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "hydra.chart" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "hydra.name" . }}
+  template:
+    metadata:
+      name: {{ include "hydra.fullname" . }}
+      labels:
+        app.kubernetes.io/name: {{ include "hydra.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/managed-by: {{ .Release.Service }}
+        helm.sh/chart: {{ include "hydra.chart" . }}
+    spec:
+{{- if .Values.postgresql.enabled }}
+      initContainers:
+      - name: db
+        image: {{ printf "%s:%s" .Values.image .Values.imageTag | quote }}
+        imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+        args:
+        - migrate 
+        - sql
+        - -e
+        env:
+{{ include "database" . | indent 8 }}
+{{ include "settings" . | indent 8 }}
+{{- end }}
+      containers:
+      - name: hydra
+        image: {{ printf "%s:%s" .Values.image .Values.imageTag | quote }}
+        imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+        args:
+        - serve
+        - all
+        - --config=/hydra/config.yaml
+        ports:
+        - name: service
+          containerPort: 4444
+        env:
+{{ include "database" . | indent 8 }}
+{{ include "settings" . | indent 8 }}
+        volumeMounts:
+          - name: config
+            mountPath: /hydra
+        livenessProbe:
```

Comment:
This probe is the reason for CrashLoop, seems that `/health` is missing on the server, which causes `time="2019-04-11T08:38:39Z" level=info msg="completed handling request" measure#http://localhost:4444.latency=250542 method=GET remote="172.17.0.1:38242" request=/health status=404 text_status="Not Found" took="250.542µs"`

Reply:
Ahh ofc! Thanks, I've pushed a quick fix which should work now. :)